lib/proxy_api/resource.rb

```diff
@@ -122,6 +122,9 @@ def ssl_auth_params
         :ssl_ca_file      =>  ca_cert,
         :verify_ssl       =>  OpenSSL::SSL::VERIFY_PEER
       }
+    rescue StandardError => exception
+      Foreman::Logging.exception("Unable to read SSL certification or key for proxy communication", exception)
+      raise Foreman::Exception.new(N_("Unable to read SSL certification or key for proxy communication: %s"), exception.message)
```

Comment:
Perhaps add info about which settings should be checked? something like `Please check settings for ssl_certificate, ssl_ca_file and ssl_priv_key and ensure they are readable by the foreman user.`?

Reply:
Done.